packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -147,7 +162,7 @@ describe('quick filter tests', () => {
   function makeFilterColumn(type = 'string'): Column {
     const filter = makeFilter();
     const column = IrisGridTestUtils.makeColumn('test placeholder', type, 13);
-    column.filter = jest.fn(() => filter);
+    column.filter = jest.fn(() => filter as FilterValue);
     return column;
   }
```

Comment:
Much like `MockFilterCondition` is defined above, define `MockColumn` here - that will allow you to get rid of all the casting you have elsewhere. E.g.
```
  type MockColumn = Omit<Column, 'filter'> & { filter(): MockFilter };
  function makeFilterColumn(type = 'string'): MockColumn {
    const filter = makeFilter();
    const column = IrisGridTestUtils.makeColumn('test placeholder', type, 13);
    column.filter = jest.fn(() => filter);
    return column as MockColumn;
  }
```